.github/workflows/api.yaml

```diff
@@ -0,0 +1,30 @@
+name: API CI
+
+on:
+  push:
+    branches: [ dev ]
```

Comment:
This should work on all branches and not just dev. Please fix this for the other job as well.

Reply:
Alright.